app/controllers/sync_controller.rb

```diff
@@ -5,12 +5,10 @@ class SyncController < ApplicationController
   before_action :skip_authorization
 
   def sync_all
-    PullUpdates.new.perform
     head :ok
   end
 
   def sync_individual
-    UpdateSubmission.call(params[:data].permit!.to_h)
```

Comment:
Just to try not to break stuff - the next PR removes them, but I'll only merge that once the app store one that stops calling these endpoints is merged, otherwise it'll start 404ing in the interim